docs/qe-available-tasks/sealights/python-instrumentation/python-instrumentation.md

```diff
@@ -0,0 +1,108 @@
+# Sealights Python Instrumentation Tekton Task
+
+This Tekton task automates the process of instrumenting python code with Sealights for quality analytics and testing. It retrieves the source code from a trusted artifact, installs Python Sealights agent, configures the app for sealights using vars from your pipeline run, scans all .py files that are not in the --exclude step above and reports scan to sealights, and stores results to be used later on in testing. The task can be triggered by either Pull Request or other events.
```

Comment:
btw what do you mean by `--exclude step above` - I guess that's copy-pasted from the Tekton Task's script?